CHANGES.txt

```diff
@@ -95,21 +96,20 @@ Build and continuous integration
 * Add a CI job for testing against traits 6.0 (#1108)
 * Update appveyor config to also build on macos (#1160)
 * Add flake8 task to CI with excludes (#1222)
-* Add a new shell command to etstool.py (#1244)
 * Add unittest for ui_traits (#1057)
 * Mention additional dependencies for demo examples (#1147)
+* Explicitly require a minimum version for Traits (#1323)
 
 Maintenance and code organization
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* Use collections.abc instead of collections (#1103, #1129)
-* Explicitly require a minimum version for Traits (#1323)
 * Relax constrain on PySide2 version (#1146)
 * Update edm version in travis and appveyor config files (#1049)
 * Resolve warnings in tests from QItemSelectionModel with Qt5 (#1041)
 * Improve how we capture and re-raise errors in GUI tests (#1099)
-* Replace logger.warn with logger.warning (#1165)
 * Replace screen metrics code with Pyface implementation (#1322)
+* Fix QDateTime deprecation warning (#1310)
```

Comment:
Why was this deprecation warning separated from the others in the fixes section?  Not necessarily a suggested change, just trying to understand the reasoning

Reply:
Because that particular warning comes from TraitsUI test suite, so users of TraitsUI should not have seen it.

Reply:
I updated this item in a subsequent commit to mention the testing context.

Reply:
I see, that makes sense.  Thanks!